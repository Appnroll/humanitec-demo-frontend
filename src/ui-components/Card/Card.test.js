import React from 'react'
import renderer from 'react-test-renderer'
import FjCard from 'ui-components/Card/Card'
import 'jest-styled-components'

test('card test', () => {
    const tree = renderer.create(<FjCard />).toJSON()
    expect(tree).toMatchSnapshot()
})
