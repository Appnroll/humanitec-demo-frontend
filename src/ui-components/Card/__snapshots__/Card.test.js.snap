// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card test 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin: 0.25rem;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c0 .card {
  box-shadow: 0 0.125rem 0.1875rem 0 rgba(0,0,0,0.12);
  background-color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-sizing: border-box;
  cursor: pointer;
}

.c0 .card__content {
  padding: 1.25rem;
}

.c0 .card__body {
  max-height: 0;
  height: auto;
  overflow: hidden;
  -webkit-transition: max-height linear 2s;
  transition: max-height linear 2s;
}

.c0 .card__body .card__content {
  border-top: 0.0625rem solid #e6e6e6;
}

.c0 .card:hover {
  background-color: #f3f3f3;
}

<div
  className="c0"
>
  <div
    className="card"
    onClick={[Function]}
  >
    <div
      className="card__header"
    >
      <div
        className="card__content"
      />
    </div>
    <div
      className="card__body"
    >
      <div
        className="card__content"
      />
    </div>
  </div>
</div>
`;
